go.mod

```diff
@@ -61,7 +61,9 @@ require (
 	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp v1.3.0
 	go.opentelemetry.io/otel/sdk v1.3.0
 	go.opentelemetry.io/otel/trace v1.3.0
+	golang.org/x/crypto v0.0.0-20220315160706-3147a52a75dd // indirect
 	golang.org/x/net v0.0.0-20211216030914-fe4d6282115f
+	golang.org/x/oauth2 v0.0.0-20210819190943-2bc19b11175f // indirect
```

Comment:
This is probably because the original PR was created _before_ the `go.mod` was re-formatted with Go 1.17. Could you move these back to the second `require` section below?

Reply:
Should be done now

Reply:
Thanks!